internal/meta_client.go

```diff
@@ -19,7 +19,7 @@ type MetaClientMock struct {
 	CreateUserFn                        func(name, password string, admin bool) (*meta.UserInfo, error)
 
 	DatabaseFn  func(name string) *meta.DatabaseInfo
-	DatabasesFn func() ([]meta.DatabaseInfo, error)
+	DatabasesFn func() []meta.DatabaseInfo
```

Comment:
This looks unrelated to this PR?  Either update PR comments or move to different PR?